internal/pkg/logreceiver/logreceiver.go

```diff
@@ -74,6 +75,19 @@ func (srv *Server) Serve() error {
 		remoteAddr, _ := netip.AddrFromSlice(srcAddr.IP)
 		remoteAddress := conn.RemoteAddr().String()
 
+		if !srv.handler.HasLink(remoteAddr) {
+			srv.logger.Warn(
+				"ignoring log handler connection attempt from the remote address: machine is not accepted yet",
+				zap.String("addr", remoteAddr.String()),
+			)
+
+			if err = conn.Close(); err != nil {
+				return err
+			}
+
+			continue
+		}
```

Comment:
This would take care of unaccepted bare metal machines spamming the logs with errors, right?

Reply:
yep